components/list/list-item-nav-button-mixin.js

```diff
@@ -111,6 +113,20 @@ export const ListItemNavButtonMixin = superclass => class extends ListItemButton
 		super._onButtonClick(e);
 	}
 
+	#handleFocusIn() {
+		requestAnimationFrame(() => {
+			const activeElement = getComposedActiveElement();
+			const parentListItem = findComposedAncestor(activeElement, (node) => node.role === 'row' || node.role === 'listitem');
+			if (parentListItem && parentListItem === this) {
+				this._focusingElem = true;
+			}
+		});
+	}
+
+	#handleFocusOut() {
+		this._focusingElem = false;
+	}
+
```

Comment:
This is so that we can use a separate property, `_focusingElem`, to check if the specific element is receiving focus so that we can apply styles only when `_focusingElem` is true. Regular list items apply `_focusing` to the parent list items as well.

If we end up changing things so that regular list items also no longer apply focus styles to the parent item when a child is focused then I'd imaging we'd move this logic into `list-item-mixin`.